index.md

```diff
@@ -6,13 +6,40 @@ nav_order: 1
+[DENT](https://dent.dev/) is an open-source network operating system (NOS) designed for decentralized enterprises. It leverages familiar Linux commands for network configuration, providing an abstracted user experience. With DENT, users can configure switches from different vendors without the need to learn specific commands associated with each.
 
-In the first release, Arthur, DENT utilizes the Linux 5.6 Kernel to leverage switchDev, enabling the replacement of proprietary SDKs and NOSes with a standard open Linux interface. It also supports existing Linux toolchains.
+![SwitchDev based Open Networking Operating System (NOS)](Images/DENT-Architecture.png)
 
-With the second release, Beeblebrox, DENT now includes secure scaling with IPv6 and NAT.
+## Releases
 
-In the third release, Cynthia, developers promise a rapid release cycle that will implement Traffic Control Persistence, QoS, IGMP Policing, and Egress Policing.
+The ["Arthur" release (v1)](https://github.com/dentproject/dentOS/wiki/1.0-Release-notes), launched in Q4 2020, introduced fundamental features such as L2 Bridge (vlan aware), IPv4 forwarding, Static Routing, Dynamic Routing – BGP, and more. It provided a robust foundation for network configuration and management, enabling seamless integration into decentralized enterprise environments.
+
+In Q1 2022, the ["Beeblebrox" release (v2)](https://github.com/dentproject/dentOS/milestone/2) brought significant enhancements to DENT. Notable additions included NAT-44 and PT, NAT on pure Layer-3 port, support for configuring 802.1q interfaces, PoE controller integration, Dynamic Routing support, and Scale & Performance Improvements. These advancements fortified DENT's capabilities in handling diverse networking requirements with improved efficiency and scalability.
+
+The "Cynthia" release series marked a significant milestone for DENT. With the [v3.0 release](https://github.com/dentproject/dentOS/milestone/3) in Q1 2023, DENT introduced TC Persistence (Petunia), a new Kernel 5.15 (LTS), and support for IEEE 802.1x, QoS, IPv6, IGMP Snooping, and Egress Policer. These features enhanced DENT's security, performance, and compatibility, positioning it as a robust solution for distributed enterprise edge networking.
+
+The subsequent updates in the "Cynthia" series further solidified DENT's capabilities. The [v3.1 release](https://github.com/dentproject/dentOS/releases/tag/v3.1) in Q2 2023 extended test coverage for QoS, ACL, and Bridging/VLAN, ensuring comprehensive validation of these critical features.
+
+Additionally, the [v3.2 release](https://github.com/dentproject/dentOS/releases/tag/v3.1) in Q3 2023 included an update to Debian 10, introduced DentOS VM for enhanced virtualization support, and extended test coverage for IPV6, IGMP Snooping, Egress Policer, Storm Control, and Devlink integration. These updates reflected DENT's commitment to continuous improvement and adaptation to evolving networking needs.
```

Comment:
The link is wrong. It should have been https://github.com/dentproject/dentOS/releases/tag/v3.2

Reply:
It is still wrong. Please make the following change
from 
    Additionally, the [v3.2 release](https://github.com/dentproject/dentOS/releases/tag/v3.1) in Q3 2023
to 
    Additionally, the [v3.2 release](https://github.com/dentproject/dentOS/releases/tag/v3.2) in Q3 2023